exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -57,19 +57,19 @@
               <th>Control Port</th>
               <th>Data Port</th>
               <th>Version</th>
+              <th>Status</th>
             </tr>
           </thead>
           <tbody>
             <#assign i = 1>
             <#list model.getDrillbits() as drillbit>
-              <tr>
+              <tr id="row-${i}">
                 <td>${i}</td>
-                <td>${drillbit.getAddress()}
-                  <#if drillbit.isCurrent()>
+                <td id="address" >${drillbit.getAddress()}<#if drillbit.isCurrent()>
                     <span class="label label-info">Current</span>
                   </#if>
                 </td>
-                <td>${drillbit.getUserPort()}</td>
+                <td id="port" >${drillbit.getUserPort()}</td>
```

Comment:
Can't us an id here; this is a list of many Drillbits and ids in HTML must be unique (that is, after all, what "identifier" means...)

If this is form formatting, use a class. If for identification, append a suffix: "id-1" or "id-xxx-xxx-xxx-xxx:xx".

Reply:
Yes, true. But for every row I have a different ID ("row-1") and I'm accessing "port"(child) of that row.  This was the entire row is unique and need not assign  a new unique id for every element in that row. Am I missing something here?